proto/jobworker/v1/service_api.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package jobworker.v1;
+
+option go_package = "github.com/tjper/teleport/proto/jobworker/v1";
+
+// JobWorkerService provides methods that allow clients to start and interact
+// with arbitrary commands running on the service's host.
+service JobWorkerService {
+  rpc Start(StartRequest) returns (StartResponse){}
+  rpc Stop(StopRequest) returns (StopResponse){}
+  rpc Status(StatusRequest) returns (StatusResponse){}
+  rpc Output(OutputRequest) returns (stream OutputResponse){}
+}
+
+// StartRequest specifies job details for JobWorkerService.Start.
+message StartRequest {
```

Comment:
How do I provide arguments to a command?

Reply:
Same as question above.

Reply:
I had planned on passing the entire command name and its arguments in a single field. I can also, break things into separate fields like this:

```
message Command {
  string name = 1;
  repeated string args = 2;
}
```
However I don't quite see the advantage immediately, but maybe I'm missing some case I'd run into during my implementation.

Reply:
I asked because it wasn't overly obvious if you just missed the arguments or if they were supposed to be passed along in the command string.

Reply:
I'll update the comment to include this detail.

Reply:
Sounds good, I've updated the proto definition.